src/l1-resources/search-index/index.ts

```diff
@@ -71,7 +71,7 @@ export interface CfnSearchIndexProps {
    *
    * @schema CfnSearchIndexProps#Mappings
    */
-  readonly mappings: ApiAtlasFtsMappingsViewManual;
+  readonly mappings?: ApiAtlasFtsMappingsViewManual;
```

Comment:
we should update the related test as well: https://github.com/mongodb/awscdk-resources-mongodbatlas/blob/main/test/l1-resources/search-index/index.test.ts

Reply:
thx, added